hedera-node/hedera-addressbook-service-impl/src/test/java/com/hedera/node/app/service/addressbook/impl/test/handlers/AddressBookTestBase.java

```diff
@@ -342,11 +342,16 @@ protected Node createNode() {
                 .build();
     }
 
-    protected Key mockPayerLookup(Key key, AccountID contextPayerId, ReadableAccountStore accountStore) {
+    protected void mockAccountLookup(Key key, AccountID contextPayerId, ReadableAccountStore accountStore) {
         final var account = mock(Account.class);
         given(account.key()).willReturn(key);
         given(accountStore.getAccountById(contextPayerId)).willReturn(account);
-        return key;
+    }
+
+    protected void mockAccountKeyOrThrow(Key key, AccountID contextPayerId, ReadableAccountStore accountStore) {
+        final var account = mock(Account.class);
+        given(account.keyOrThrow()).willReturn(key);
+        given(accountStore.getAccountById(contextPayerId)).willReturn(account);
     }
```

Comment:
Another nit: any reason not to include line 353 inside of `mockAccountLookup` instead of creating a separate but almost identical method?